frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -117,6 +132,21 @@ const UserProfilePage = () => {
       setAdditionalExperiences(
         parseUserBackground(user?.additionalExperiences),
       );
+      setTempFullName(`${user?.firstName} ${user?.lastName}`);
+      setTempEmail(user?.email!);
+      const tempAdditionalExperiences = parseUserBackground(
+        user?.additionalExperiences,
+      );
+      setTempEducationalQualification(
```

Comment:
currently if the language experience or education fields are not empty and the user clicks cancel, it resets the fields to `""`. i think the fix is to call `setEducationQualification` and `setLanguageExperience` here also